src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationFunctionsTests.cs

```diff
@@ -364,6 +368,112 @@ public void DontCopyDerivedRecordValuesTest(string expression)
             Assert.Equal("x", fileObjectRecordValue.SomeProperty);
         }
 
+        [Theory]
+        [InlineData("Patch(t, First(t), {Value:1})")]
+        public void MutationPFxV1Disabled(string expression)
+        {
+            var engine = new RecalcEngine(new PowerFxConfig(Features.None));
+            var t = FormulaValue.NewTable(RecordType.Empty().Add(new NamedFormulaType("Value", FormulaType.Decimal)));
+
+            engine.Config.SymbolTable.EnableMutationFunctions();
+            engine.UpdateVariable("t", t);
+
+            var check = engine.Check(expression, options: new ParserOptions() { AllowsSideEffects = true });
+
+            // Compilation will be successful, but the function will not be executed.
+            // This is because PA depends on the CheckType to determine if the function is valid.
+            Assert.True(check.IsSuccess);
+
+            var evaluator = check.GetEvaluator();
+
+            // Runtime exception
+            Assert.ThrowsAsync<InvalidOperationException>(async () => await evaluator.EvalAsync(CancellationToken.None).ConfigureAwait(false)).ConfigureAwait(false);
+        }
+
+        [Theory]        
+        [InlineData("Patch(t1, {accountid:GUID(\"00000000-0000-0000-0000-000000000001\"),name:\"Mary Doe\"});First(t1).name", "Mary Doe")]
+        [InlineData("Patch(t1, {accountid:GUID(\"00000000-0000-0000-0000-000000000001\"),address1_city:\"Seattle\"});First(t1).name", "John Doe")]
+        [InlineData("Patch(t1, {accountid:GUID(\"00000000-0000-0000-0000-000000000003\"),name:\"Microsoft Corporation\",address1_city:\"Seattle\"});Last(t1).name", "Microsoft Corporation")]
```

Comment:
>Last(t1).name [](http://example.com/codeflow?start=148&length=13)

Can you also add the number of elements in the expression, so that we will show that it added one? Something like `$"{CountRows(t1) - {Last(t1).name}"`?